packages/config/src/layer2s/zkfair.ts

```diff
@@ -0,0 +1,37 @@
+import { ProjectId } from '@l2beat/shared-pure'
+
+import { CONTRACTS, TECHNOLOGY, UPCOMING_RISK_VIEW } from './common'
+import { Layer2 } from './types'
+
+export const zkfair: Layer2 = {
+  isUpcoming: true,
+  type: 'layer2',
+  id: ProjectId('zkfair'),
+  display: {
+    name: 'ZKFair',
+    slug: 'zkfair',
+    description:
+      'ZKFair is the first community Validium based on Polygon CDK and Celestia DA, championing fairness.',
+    purpose: 'Universal',
+    category: 'Validium',
+    dataAvailabilityMode: 'NotApplicable',
+    provider: 'Polygon',
+    links: {
+      websites: ['https://zkfair.io/'],
+      apps: [],
+      documentation: ['https://docs.zkfair.io/'],
+      explorers: [],
+      repositories: [],
+      socialMedia: ['https://twitter.com/ZK_fair/'],
```

Comment:
```suggestion
      socialMedia: ['https://twitter.com/ZK_fair'],
```

Reply:
I have addressed it. thank you very much.